commands/Tools/contact.js

```diff
@@ -0,0 +1,37 @@
+exports.run = async (client, msg, [type, ...content]) => {
+  // Determines which type of contact and sends to the appropriate channel.
+  let output = "";
+  if (type === "bug") {
+    output += "Bug Report";
+  } else if (type === "idea") {
+    output += "New Idea";
+  } else {
+    output += "New Message";
+  }
+  this.channel.send(`${output} From: ${msg.author} in ${msg.guild}\n\n ${content.toString().replace(/,/g, " ")}`);
+};
+
+exports.conf = {
+  enabled: true,
+  runIn: ["text"],
+  aliases: ["report"],
+  permLevel: 0,
+  botPerms: [],
+  requiredFuncs: [],
+  cooldown: 0,
+};
+
+exports.help = {
+  name: "contact",
+  description: "Need to contact the developers? Reporting a bug? Sending a suggestion?",
+  usage: "<bug|idea|message> <content:str> [...]",
+  usageDelim: " ",
+  extendedHelp: "This is a cool command that will let you use it in different ways. If you do contact bug you can send a bug report. If you do contact idea you can send it to the suggestion pile. If you do message you can send us a private message to the developers. Please note it is extremely helpful to leave a discord invite link so incase we can't understand we can contact you.",
+};
+
+exports.channel = null;
+
+exports.init = (client) => {
+  // Insert channel ID of the channel you would like for the users to contact you on.
+  this.channel = client.channels.get("CHANNEL_ID");
```

Comment:
Maybe use a config option for this? I feel that would be better than hardcoding a channel ID in and would make it easy to change later, if necessary.

Reply:
I agree: `client.config.supportChannel` 👍

Reply:
Would you guys mind sending me a link to the doc page where I can learn to add a config option? I've been looking through it and the only options I can add is in settings gateway but that's per guild.

Reply:
Any variables passed to Komada's startup will be available on `client.config`
Example:
```js
new Komada({
  prefix: blah,
  ..more here,
  myDefaultSupportChannel: 'someID',
  }).login(token);

// You now have the channel ID available from `client.config.myDefaultSupportChannel`

Reply:
But how do I pass a variable to already existing Komada startup in a Komada piece? this piece has to be standalone and i have no way to force a user to edit their komada configurations. @UnseenFaith